qbraid_qir/qasm3/profiles/core.py

```diff
@@ -0,0 +1,223 @@
+
```

Comment:
Please add the headers in all new Python files. See [reference](https://github.com/qBraid/qbraid-qir/blob/main/qbraid_qir/qasm3/convert.py#L1)